src/cli/utils.cpp

```diff
@@ -235,9 +239,10 @@ class Base64_Encode final : public Command
 
       void go() override
          {
-         this->read_file(get_arg("file"),
-                         [&](const uint8_t b[], size_t l) { output() << Botan::base64_encode(b, l); },
-                         768);
+         this->read_file(get_arg("file"), [&](const uint8_t b[], size_t l)
+            {
+            output() << Botan::base64_encode(b, l);
+            }, 768);
```

Comment:
Not an improvement in readability, instead of having the 3 args to `read_file` be one per line, it smears the callback definition across 4 lines, 2 of which are shared with the other arguments.

astyle probably doesn't have logic for C++11 lambdas, not sure what the best fix here is.